### ISSUE TITLE
Digitigrade legs feet slot is greyed out, as if you can't equip legwear

### ISSUE THREAD

**Issue by SapphoQueer** [Bubberstation/Bubberstation#2403]
### Issue Summary:

![image](https://github.com/user-attachments/assets/5be2e844-28c8-4ada-98f9-dcd397523d92)
It's grey

### Round ID:

5943

### Test Merge Information:

The following pull requests are currently test merged:
[#1545: 'Public Logging Update' by nevimer at commit 687d0d08e5](https://github.com/Bubberstation/Bubberstation/pull/1545)
[#1919: '[TM ONLY] Unfucking disease outbreaks/transmission' by lessthnthree at commit c9df7c9c1f](https://github.com/Bubberstation/Bubberstation/pull/1919)
[#2116: '[NOT READY FOR MERGE] [READY FOR REVIEW] Storyteller 2.2' by Majkl-J at commit b16b530d1c](https://github.com/Bubberstation/Bubberstation/pull/2116)
[#2199: 'Space MP5 - Redux' by Offwrldr at commit dd94c677fc](https://github.com/Bubberstation/Bubberstation/pull/2199)
[#2193: 'Succumbing now alerts everyone around you that you gave up on life.' by StrangeWeirdKitten at commit 5e60bb8a88](https://github.com/Bubberstation/Bubberstation/pull/2193)
[#2201: 'Investigate Stat Panel as a Stutter cause' by nevimer at commit 60878d6988](https://github.com/Bubberstation/Bubberstation/pull/2201)
[#2265: 'Persistence Base (A new offstation syndicate map) (Should be ready for test merge)' by shellspeed1 at commit 494e163a68](https://github.com/Bubberstation/Bubberstation/pull/2265)
[#2251: 'Adds a new emote: Bonk' by xPokee at commit c272baf451](https://github.com/Bubberstation/Bubberstation/pull/2251)
[#2291: 'Saddles, saddlebags, calvary' by nikothedude at commit 1e5dd05d0c](https://github.com/Bubberstation/Bubberstation/pull/2291)
[#2334: 'Renames all mentions of Solfed to TerraGov' by Arturlang at commit 23fbf5cf11](https://github.com/Bubberstation/Bubberstation/pull/2334)
[#2335: 'Added 'Nyamagotchi'' by Middiwiddi at commit c6ce817be6](https://github.com/Bubberstation/Bubberstation/pull/2335)
[#2356: 'Mouseover Highlights while holding shift' by nevimer at commit 57a59b07f0](https://github.com/Bubberstation/Bubberstation/pull/2356)
[#2206: '(Port) MKUltra - Mind Control' by xPokee at commit f725ee3297](https://github.com/Bubberstation/Bubberstation/pull/2206)

### Reproduction Steps:

1. Have digitigrade legs
2. it's grey

**Comment by lessthnthree** [comment]
It's grey, but the slot still works.

https://github.com/user-attachments/assets/f1461bd7-5c53-47da-8b3f-c2f21b2101c2

**Comment by SapphoQueer** [comment]
> It's grey, but the slot still works.

Yes I did mention this in the original. I just hate that it's grey.
